engine/src/main/java/org/terasology/config/AudioConfig.java

```diff
@@ -31,39 +31,63 @@
 
     private transient PropertyChangeSupport propertyChangeSupport = new PropertyChangeSupport(this);
 
+    /**
+     * @return Returns a float of the sound volume
+     */
     public float getSoundVolume() {
         return soundVolume;
     }
 
+    /**
+     * @param soundVolume Sets the sound volume starting from the previous value and going to the soundVolume passed in
```

Comment:
...`@param soundVolume the sound volume starting from the previous value and going to the soundVolume passed in` is fine here and in every other setter 🙂